Facebook-post-likes-comment-shares.py

```diff
@@ -9,7 +16,7 @@ class FaceBookBot():
             'email': username,
             'pass': password
         }
-    post_ID = ""
+    post_ID = "2618383005050896"
 
     def parse_html(self, request_url):
```

Comment:
It seems like `parse_html` is not logging into Facebook successfully. We might need to change to Selenium to log into Facebook rather than using `requests`.

Reply:
I'm new on python can you share the new selenium code ? :)

Reply:
Hi there. I made a video showing you how to use Selenium to log into Facebook: https://youtu.be/ZizelXeaS1Y 
The code that is used can be found here: https://gist.github.com/adriaan90/83c09d85ffa9352d0cc013ee3e208d49
Hope it helps 👍

Reply:
thank you so much :)